util/build_configs/cray-internal/chapel_package-cray.bash

```diff
@@ -188,7 +188,9 @@ fi
 # Generate modulefile, w versions
 
 log_debug "Generate modulefile-$pkg_version ..."
-$cwd/generate-modulefile.bash > "$rpmbuild_dir/modulefile-$pkg_version"
```

Comment:
Can remove `generate-modulefile.bash` now?

Reply:
Oh yeah, that would be a good follow-up. I was so oriented toward getting the produced modulefile right I forgot that part.